plugins/linux/bccbpf/Makefile

```diff
@@ -1,7 +1,7 @@
 CC=gcc
 
-all: iolatency.elf
+all: bpf.elf iolatency.elf
 
-iolatency.elf: iolatency.c
-	$(CC) iolatency.c -o iolatency.elf
-	chmod +s iolatency.elf
+%.elf : %.c
+	$(CC) $< -o $@
+	chmod +s $@
```

Comment:
Creating a setuid executable is security risk. Shall we automate this step, or leave it to the user to do this manually.